finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/H2Transport.scala

```diff
@@ -37,5 +37,7 @@ object H2Transport {
     def goAway(err: GoAway, deadline: Time): Future[Unit]
 
     final def goAway(err: GoAway): Future[Unit] = goAway(err, Time.Top)
+
+    def ping(): Future[Unit]
```

Comment:
The semantics of this are a bit unexpected, so this should definitely have a comment.  I think the returned future is satisfied when the ping is written, not when the ack is received as one might expect.

Reply:
Consider renaming the method to `sendPing()`, which may help to clarify this.